dotnet/samples/KernelSyntaxExamples/Example14_SemanticMemory.cs

```diff
@@ -35,13 +36,19 @@ public static async Task RunAsync()
          * need to worry about embedding generation.
          */
 
-        var kernelWithACS = Kernel.Builder
-            .WithLoggerFactory(ConsoleLogger.LoggerFactory)
+        var kernel = Kernel.Builder
             .WithOpenAITextEmbeddingGenerationService("text-embedding-ada-002", TestConfiguration.OpenAI.ApiKey)
```

Comment:
Does Kernel still need the WithOpenAITextEmbeddingGenerationService extension method?

Reply:
In theory Kernel doesn't need it, but it's still possible to register multiple embedding generation services and then get a specific service with `kernel.GetService<ITextEmbeddingGeneration>(name)`. Removing `.GetService` capability from Kernel and related extension methods should be part of separate PR I think.